### PR TITLE
[2.19.x] DDF-UI-318 add validation for relative time input

### PR DESCRIPTION
**Pull request by andrewzimmer**
#### What does this PR do?
add validation for relative time input, a value that is too large will break the query

#### Who is reviewing it? 
@cassandrabailey293 @hayleynorton @zta6 @abel-connexta 

#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining 
@mojogitoverhere

#### How should this be tested?
- build / install / run
- attempt to enter a large value in the relative time search, verify inline error message appears
- attempt search with invalid value, verify search is blocked and a relevant message appears

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/utils/validation/validation.tsx [hunk 1]
```diff
@@ -172,6 +186,18 @@ function hasPointError(point: any[]) {
   return point[0] > 180 || point[0] < -180 || point[1] > 90 || point[1] < -90
 }
 
+function getDateErrors(filter: any): Set<string> {
+  const errors = new Set<string>()
```
Comment (zta6) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/utils/validation/validation.tsx:190]:
```suggestion
  const errors = new Set<string>()
  const value = deserialize(filter.value)
  if (value !== undefined && relativeTimeError(value)) {
    errors.add('relative time exceeds max value')
  }
  return errors
}
```

This should work too. A bit shorter but also a bit less easy to read, your call.

**Comment by bdeining** [comment]
build now

**Comment by andrewzimmer** [comment]
build now

**Comment by bdeining** [comment]
hero looks good

**Pull request merged**